docs/source/running.rst

```diff
@@ -0,0 +1,20 @@
+.. _running:
+
+
+====================
+Running the Notebook
+====================
+
+After you have installed the Jupyter Notebook on your computer, you are ready
+to run the Notebook. To do this, run the command::
```

Comment:
Run the command where ? People that will land on this page directly might be lost. 
Anaconda will also most likely have a launcher.